app/models/concerns/hmis/structure/base.rb

```diff
@@ -13,6 +13,13 @@ module HMIS::Structure::Base
     scope :delete_pending, -> do
       where.not(pending_date_deleted: nil)
     end
+
+    def imported_item_type(importer_log_id)
```

Comment:
This seems like it should be getting used somewhere?

Reply:
Line 35 in app/controllers/hmis_controller.rb